services/datcore-adapter/tests/unit/test_utils_app_data.py

```diff
@@ -0,0 +1,47 @@
+# pylint:disable=unused-variable
```

Comment:
why is this file commented out?

Reply:
hmm I am not sure what you mean there? it does not appear commented out in my branch... or maybe I'm missing something

Reply:
Never mind, looks good this morning.